templates/config/nginx/default.conf.mustache

```diff
@@ -0,0 +1,185 @@
+		set $skip 1;
+	}
+	# Don't use the cache for logged in users or recent commenter or customer with items in cart
+	if ($http_cookie ~* "comment_author|wordpress_[a-f0-9]+|wp-postpass|wordpress_no_cache|wordpress_logged_in|woocommerce_items_in_cart") {
+		set $skip 1;
+	}
+	# Use cached or actual file if they exists, Otherwise pass request to WordPress
+	location / {
+		try_files $uri $uri/ /index.php?$args;
+	}
+
+	location /redis-fetch {
+		internal  ;
+		set  $redis_key $args;
+		redis_pass  ee_redis:6379;
+	}
+	location /redis-store {
+		internal  ;
+		set_unescape_uri $key $arg_key ;
+		redis2_query  set $key $echo_request_body;
+		redis2_query expire $key 14400;
+		redis2_pass  ee_redis:6379;
+	}
+
+	location ~ \.php$ {
+		#  add_header Cache-Control "max-age=0, no-cache, no-store, must-revalidate";
+		set $key "nginx-cache:$scheme$request_method$host$request_uri";
+		try_files $uri =404;
+
+		srcache_fetch_skip $skip;
+		srcache_store_skip $skip;
+
+		srcache_response_cache_control off;
+
+		set_escape_uri $escaped_key $key;
+
+		srcache_fetch GET /redis-fetch $key;
+		srcache_store PUT /redis-store key=$escaped_key;
+
+		more_set_headers 'X-SRCache-Fetch-Status $srcache_fetch_status';
+		more_set_headers 'X-SRCache-Store-Status $srcache_store_status';
+
+		include fastcgi_params;
+		fastcgi_pass php:9000;
+	}
+
+	{{/include_redis_conf}}
+
+	{{#include_wpsubdir_conf}}
+	# WPSUBDIRECTORY NGINX CONFIGURATION
+	if (!-e $request_filename) {
+
+		# Redirect wp-admin to wp-admin/
+		rewrite /wp-admin$ $scheme://$host$uri/ permanent;
+
+		# Redirect wp-* files/folders
+		rewrite ^(/[^/]+)?(/wp-.*) $2 last;
+
+		# Redirect other php files
+		rewrite ^(/[^/]+)?(/.*\.php) $2 last;
+	}
+	{{/include_wpsubdir_conf}}
+
+	{{#include_php_conf}}
+	# PHP NGINX CONFIGURATION
+	location / {
+		try_files $uri $uri/ /index.php?$args;
+	}
+	location ~ \.php$ {
+		try_files $uri =404;
+		fastcgi_param  SCRIPT_FILENAME  $document_root$fastcgi_script_name;
+		include fastcgi_params;
+		fastcgi_pass php:9000;
+	}
+	{{/include_php_conf}}
+
+	{{! wpcommon.conf }}
+	# WordPress COMMON SETTINGS
+	# Limit access to avoid brute force attack
+	location = /wp-login.php {
+		limit_req zone=one burst=1 nodelay;
+		include fastcgi_params;
+		fastcgi_pass php:9000;
+	}
+	# Disable wp-config.txt
+	location = /wp-config.txt {
+		deny all;
+		access_log off;
+		log_not_found off;
+	}
+	# Disallow php in upload folder
+	location /wp-content/uploads/ {
+		location ~ \.php$ {
+			#Prevent Direct Access Of PHP Files From Web Browsers
+			deny all;
+		}
+	}
+	{{! /wpcommon.conf }}
+
+	{{! locations.conf }}
+	# NGINX CONFIGURATION FOR COMMON LOCATION
+	# Basic locations files
+	location = /favicon.ico {
+		access_log off;
+		log_not_found off;
+		expires max;
+	}
+
+	location = /robots.txt {
+	# Some WordPress plugin gererate robots.txt file
+	# Refer #340 issue
+	return 200 'User-agent: *\nDisallow: /';
```

Comment:
@kirtangajjar This line will deindex site from the search engines! 😨 
Nginx won't use robotos.txt files on disk.
Please remove this line.

Reply:
@mbtamuli this need to be changed